File: src/main/java/com/bindstone/kuljetus/domain/LibelleMarqueCount.java
package com.bindstone.kuljetus.domain;

import java.util.StringJoiner;

public class LibelleMarqueCount {

    Long count;
    private String libelleMarque;

    public String getLibelleMarque() {
        return libelleMarque;
    }

    public void setLibelleMarque(String libelleMarque) {
        this.libelleMarque = libelleMarque;
    }

    public Long getCount() {
        return count;
    }

    public void setCount(Long count) {
        this.count = count;
    }

    @Override
    public String toString() {
        return new StringJoiner(",", "[", "]")
                .add("libelleMarque='" + libelleMarque + "'")
                .add("count='" + count + "'")
                .toString().concat("\n");
    }
}

File: src/main/java/com/bindstone/kuljetus/domain/LibelleMarqueCount.kt
package com.bindstone.kuljetus.domain

import java.util.*

class LibelleMarqueCount {
    var count: Long? = null
    var libelleMarque: String? = null

    override fun toString(): String {
        return StringJoiner(",", "[", "]")
                .add("libelleMarque='$libelleMarque'")
                .add("count='$count'")
                .toString() + "\n"
    }
}
